daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala

```diff
@@ -475,17 +474,17 @@ class OrderingSpec
     new util.ArrayList[X](as.asJava)
 
   private val txSeed = crypto.Hash.hashPrivateKey("SBuiltinTest")
-  private def dummyMachine = Speedy.Machine fromExpr (
-    expr = e"NA:na ()",
+  private def initMachine(expr: SExpr) = Speedy.Machine fromSExpr (
+    sexpr = expr,
     compiledPackages = PureCompiledPackages(Map.empty, Map.empty),
-    scenario = false,
     Time.Timestamp.now(),
-    Some(txSeed),
+    InitialSeeding(Some(txSeed)),
+    Set.empty,
```

Comment:
Can we please use named arguments for all the arguments here? The list is quite unwieldy already.